src/plugins/graph/models/graph-model.ts

```diff
@@ -105,21 +104,15 @@ export const GraphModel = TileContentModel
       return self.config.attributeID(place) ?? '';
     },
     getPointRadius(use: 'normal' | 'hover-drag' | 'select' = 'normal') {
-      let r = pointRadiusMax;
-      const numPoints = self.config.caseDataArray.length;
+      const r = pointRadiusMax;
       // for loop is fast equivalent to radius = max( minSize, maxSize - floor( log( logBase, max( dataLength, 1 )))
-      for (let i = pointRadiusLogBase; i <= numPoints; i = i * pointRadiusLogBase) {
-        --r;
```

Comment:
Since we don't want our circle radius to be a function of the # of points (we always want it to be 5) I removed this.